editMember.php

```diff
@@ -2,60 +2,64 @@
-      </div>
-
-      <div class="editFormInput">
-        <label for="DateofBirth" id="dob">Date of Birth</label>
-        <input type="date" pattern="\d{1,2}\/\d{1,2}\/\d{4}" name="dateofbirth" id="dateofbirth" required />
-      </div>
-
-      <div class="editFormInput">
-        <label for="email" id="em">Email</label>
-        <input type="email" id="email" name="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
-      </div>
-
-      <div class="editFormInput">
-        <label for="phonenum" id="mobilenum">Mobile Number (Optional)</label>
-        <input type="tel" name="phonenum" id="phonenum" maxlength="12" pattern="[\d\s]{8,12}" placeholder="For eg. 0400 000 000" required />
-      </div>
-
-
-
-
-      <div class="button">
-        <button class="button" id=button1 buttonType="primary" type="submit" name="submit">Save Details</button>
-      </div>
-
-      <div class="buttonCancel">
-        <button class="buttonCancel" id=button1 buttonType="secondary" type="reset" name="cancel">Cancel</button>
-      </div>
-
-
-    </form>
-  </main>
+    <?php
+  include_once("navbar.inc");
+  ?>
+    <main>
+        <form method="post" action="updateMEMBER.php" class="editMemberform">
+
+            <div class="editFormInput">
+                <h3>Edit Member Form</h3>
+            </div>
+
+            <div class="editFormInput">
+                <label for="firstname" id="fName">First Name</label>
+                <input type="text" pattern="[a-zA-Z]{2,20}" id="firstname" name="firstname" required />
+            </div>
+
+            <div class="editFormInput">
+                <label for="lastname" id="lName">Last Name</label>
+                <input type="text" pattern="[a-zA-Z]{2,20}" id="lastname" name="lastname" required />
+            </div>
+
+            <div class="editFormInput">
+                <label for="DateofBirth" id="dob">Date of Birth</label>
+                <input type="text" pattern="^\d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])$" name="dateofbirth"
+                    id="dateofbirth" required placeholder="YYYY-MM-DD" />
+            </div>
+
+            <div class="editFormInput">
+                <label for="email" id="em">Email</label>
+                <input type="email" id="email" name="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
+            </div>
+
+            <div class="editFormInput">
+                <label for="phonenum" id="mobilenum">Mobile Number (Optional)</label>
+                <input type="tel" name="phonenum" id="phonenum" maxlength="10" pattern="[\d]{10}"
+                    placeholder="For eg. 0400000000" />
+                <input type="hidden" name="member_id" id="member_id" value="1" />
+            </div>
+            <div class="button">
+                <button class="button" id=button1 buttonType="primary" type="submit" name="submit">Save Details</button>
```

Comment:
major change in editMember.php is the class name for the button.. the rest your changes can apply